solutions/security/cloud/cloud-native-vulnerability-management.md

```diff
@@ -20,15 +20,11 @@ CNVM currently only supports AWS EC2 Linux workloads.
 
 
 ::::{admonition} Requirements
-* CNVM is available to all {{ecloud}} users. On-premise deployments require an [Enterprise subscription](https://www.elastic.co/pricing).
-* Requires {{stack}} and {{agent}} version 8.8 or higher.
+* {{stack}} users: {{stack}} version 8.8 or higher and an [Enterprise subscription](https://www.elastic.co/pricing).
```

Comment:
Should this be 9.0 or higher instead? 

```suggestion
* {{stack}} users: {{stack}} version 9.0 or higher and an [Enterprise subscription](https://www.elastic.co/pricing).
```

Reply:
Nope! Feature has been available since 8.8 :)